src/Bicep.Core/Registry/AzureContainerRegistryManager.cs

```diff
@@ -27,6 +34,10 @@ public class AzureContainerRegistryManager
 
         private readonly IContainerRegistryClientFactory clientFactory;
 
+        // https://docs.docker.com/registry/spec/api/#content-digests
+        // "While the algorithm does allow one to implement a wide variety of algorithms, compliant implementations should use sha256."
```

Comment:
>algorithm [](http://example.com/codeflow?start=22&length=9)

nit - specification?

Reply:
Don't understand the question. If you're asking where, the link is the line above the quote (https://docs.docker.com/registry/spec/api/#content-digests)

Did change slightly to this:

        // From the spec: "While the algorithm does allow one to implement a wide variety of algorithms, compliant implementations should use sha256."
        // (https://docs.docker.com/registry/spec/api/#content-digests)
        private static readonly string DigestAlgorithmIdentifier = DescriptorFactory.AlgorithmIdentifierSha256;

Reply:
Ah, now I understand.  This is a direct quote from the spec, thus the double quotes.  The new change should make this clearer anyway.

Reply:
Sorry I didn't make it clear. It was literally just about the first "algorithm" word. Change looks good.